cookiecutter/utils.py

```diff
@@ -116,3 +132,30 @@ def __init__(self, environment):
 
     SimpleFilterExtension.__name__ = filter_function.__name__
     return SimpleFilterExtension
+
+
+def simple_test(test_function):
+    """
+    Register a function for use as a test extension for Jinja templates.
```

Comment:
I think flake8 is upset with this:

```suggestion
    """Register a function for use as a test extension for Jinja templates.
```